app/models/user/financial_aid/cached_award_comparison_data.rb

```diff
@@ -67,7 +67,7 @@ def get_feed_internal
               },
               {
                 description: 'Expected Family Contribution (EFC)',
-                value: isir.try(:[], 'primary_efc').partition('$ ')[1].to_f
```

Comment:
could this not be --->   value:  isir['primary_efc'].present?  ? isir['primary_efc'].partition('$ ')[1].to_f : nil much more simpler and one linder